kolibri/auth/serializers.py

```diff
@@ -45,6 +45,13 @@ class Meta:
         fields = ('id', 'username', 'full_name', 'password', 'facility', 'roles')
 
 
+class FacilityUsernameSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = FacilityUser
+        fields = ('id', 'username', )
```

Comment:
Now that I think about it, I can define the username itself to be the `id` attribute on the Model, so can remove this.